app/src/main/java/com/waseefakhtar/doseapp/extension/DateExtension.kt

```diff
@@ -15,6 +15,21 @@ fun Date.toFormattedMonthDateString(): String {
     return sdf.format(this)
 }
 
+fun Date.toFormattedYeahMonthDateString(): String {
```

Comment:
```suggestion
fun Date.toFormattedYearMonthDateString(): String {
```

Reply:
What about this? I don't see the difference

Reply:
Yeah -> Year